edx_lint/pylint/common.py

```diff
@@ -27,5 +28,5 @@ def usable_class_name(node):
     name = node.qname()
     for prefix in ["__builtin__.", "builtins.", "."]:
         if name.startswith(prefix):
-            name = name[len(prefix):]
+            name = name[len(prefix) :]
```

Comment:
Probably no point asking why black inserted a space before the colon....

Reply:
Details in https://github.com/psf/black/issues/279 .  Comes from the following part of PEP 8:

> However, in a slice the colon acts like a binary operator, and should have equal amounts on either side (treating it as the operator with the lowest priority). In an extended slice, both colons must have the same amount of spacing applied. Exception: when a slice parameter is omitted, the space is omitted.

PEP 8 basically allows for either `sequence[a:b]` or `sequence[a : b]`, but not `sequence[a: b]`.  The second of those condenses to the above when the second parameter is missing.  Granted, the code before the change is perfectly PEP 8 compliant also, I think they just didn't want to try to figure out under which conditions the space after a colon is ok to omit.